plugins/woocommerce-blocks/docs/internal-developers/blockified-templates/compatibility-layer.md

```diff
@@ -55,7 +55,7 @@ The following table shows where the hooks are injected into the page.
 | woocommerce_after_main_content            | Last block related to the Single Product Template (Product Image Gallery, Product Details, Add to Cart Form, Product Meta, Product Price, Breadcrumbs)  | after    |
 | woocommerce_sidebar                       | Last block related to the Single Product Template (Product Image Gallery, Product Details, Add to Cart Form, Product Meta, Product Price, Breadcrumbs)  | after    |
 | woocommerce_before_single_product         | First block related to the Single Product Template (Product Image Gallery, Product Details, Add to Cart Form, Product Meta, Product Price, Breadcrumbs) | before   |
-| woocommerce_before_single_product_summary | First block related to the Single Product Template (Product Image Gallery, Product Details, Add to Cart Form, Product Meta, Product Price, Breadcrumbs) | before   |
+| woocommerce_before_single_product_summary | Before first `core/post-excerpt` block                                                                                                                  | before   |
```

Comment:
```suggestion
| woocommerce_before_single_product_summary | First `core/post-excerpt` block (before `woocommerce_single_product_summary`) | before   |
```

Reply:
> LGTM, just one question: looking at the code and based on my testing, woocommerce_before_single_product_summary and woocommerce_single_product_summary work exactly the same with the only difference that woocommerce_before_single_product_summary is rendered before woocommerce_single_product_summary, no?

Exactly!

> I made a small suggestion to clarify that. But feel free to disregard it if it's not correct or you think it doesn't make things clearer. 🙂

Thanks for your suggestion! It makes sense! 🙇